### PR TITLE
Add Showcase card in Categories page

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Adds Showcase card to Categories page.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Part of https://github.com/PrestaShop/PrestaShop/issues/10434
| How to test?  | :warning: Build assets before testing. Showcase card should be available in Categories page. See screenshot below. If RTL does not work, it will be fixed in https://github.com/PrestaShop/PrestaShop/pull/12809

### Showcase card

showcase_card

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13234)

### CODE REVIEW

**Comment by matks** [comment]
Can we fix https://github.com/PrestaShop/PrestaShop/issues/12160 as well ? Harmonizing helper card and showcase card classes* ?

*Naturally we need to handle backwards compatibility for classes introduced in 1.7.5 😅

**Comment by sarjon** [comment]
> *Naturally we need to handle backwards compatibility for classes introduced in 1.7.5

`HelperCard` is not part of 1.7.5 yet, so we are good. :)

File: admin-dev/themes/new-theme/js/pages/customer/index.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ import DeleteCustomersBulkActionExtension
   from "../../components/grid/extension/action/bulk/customer/delete-customers-bulk-action-extension";
 import DeleteCustomerRowActionExtension
   from "../../components/grid/extension/action/row/customer/delete-customer-row-action-extension";
-import HelperCard from "../../components/helper-card";
+import ShowcaseCard from "../../components/showcase-card/showcase-card";
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/js/pages/customer/index.js]: single quote please

Comment (sarjon) [reply admin-dev/themes/new-theme/js/pages/customer/index.js]: Done.

**Comment by sarjon** [comment]
@matks this PR gets rid of `helper-card.js`. :)

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/showcase_card.html.twig [hunk 2]
```diff
@@ -23,16 +23,26 @@
...
-  <div class="helper-card__left helper-card__customer shape-one">
-    <img src="{{ asset('themes/new-theme/scss/img/helper-card/customer@3x.png') }}" class="img-fluid">
+{% if not isShowcaseCardClosed %}
+  <div id="categoriesShowcaseCard" class="helper-card card">
+    <div class="helper-card__left helper-card__category shape-one">
+      <img src="{{ asset('themes/new-theme/scss/img/helper-card/category@3x.png') }}" class="img-fluid img-rtl">
+    </div>
+    <div class="helper-card__right">
+      <h2>{{ 'Set categories'|trans({}, 'Admin.Catalog.Feature') }}</h2>
+      <p>{{ 'Categories are essential, as they enable you to group equivalent products. This helps customers find their way through the variety of your catalog.'|trans({}, 'Admin.Catalog.Feature') }}</p>
```
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/showcase_card.html.twig:33]: From my side, I have the following wording, _Create a menu tree to group your products and lead commercial operations so your customers can easily find what they are looking for. You won’t find such a time saver!_, I don't know why it is not the same. @TristanLDD?

Comment (matks) [reply src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/showcase_card.html.twig:33]: Warning: this page has both an Empty State and a Helper Card 😉

The Empty state appears when the listing is empty while the Helper Card appears all the time, until the user closes it (oh, I see there's going to be an issue here, when listing is empty do we display both ?).

This is the PR for the Helper Card whose text comes from https://projects.invisionapp.com/share/FQNYNAE6RDJ#/screens

File: admin-dev/themes/new-theme/js/pages/categories/index.js [hunk 3]
```diff
@@ -43,6 +43,8 @@ import textToLinkRewriteCopier from "../../components/text-to-link-rewrite-copie
 import ChoiceTree from "../../components/form/choice-tree";
 import FormSubmitButton from "../../components/form-submit-button";
 import TaggableField from "../../components/taggable-field";
+import ShowcaseCard from "../../components/showcase-card/showcase-card";
+import ShowcaseCardCloseExtension from "../../components/showcase-card/extension/showcase-card-close-extension";
```
Comment (matks) [inline admin-dev/themes/new-theme/js/pages/categories/index.js:47]: This is probably outside the scope of this PR but do we really need ShowcaseCard and ShowcaseCardCloseExtension ? Isn't it part of the same feature/behavior 🤔 ?

**Review by matks — APPROVED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/showcase_card.html.twig (hunk 2 above)
Comment (sarjon) [reply src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/showcase_card.html.twig:33]: It's a bit confusing, there are mockups in Invision & Zeplin and sometimes they differ. :/ In addition, I couldn't find Empty state for Categories, @matks do you know where I can get it from?

File: admin-dev/themes/new-theme/js/pages/categories/index.js (hunk 3 above)
Comment (sarjon) [reply admin-dev/themes/new-theme/js/pages/categories/index.js:47]: Haha, I'm sure I've asked you the same question like 6 months ago. :smile:

**Comment by TristanLDD** [comment]
Hello @sarjon, 

For this project the final mockup are the one on Zeplin (we will soon end our use of Zeplin to go full invision) I'm checking now the wording on Zeplin and I'll update if necessary.

**Comment by sarjon** [comment]
@TristanLDD is the wording correct on Categories showcase card or should it be changed?

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/showcase_card.html.twig (hunk 2 above)
Comment (sarjon) [reply src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/showcase_card.html.twig:33]: ping @LouiseBonnard @TristanLDD for wording, which one should be used?

**Comment by LouiseBonnard** [comment]
Hello @sarjon, @TristanLDD just told me that the wording on Zeplin is correct. :-)

**Comment by sarjon** [comment]
@LouiseBonnard seems that projects on Zeplin have been archived, I can't access any of them. :/

**Comment by LouiseBonnard** [comment]
@sarjon, I've just learned that the Zeplin project for PrestaShop is now closed. Here is the proper link to check the wording now: https://projects.invisionapp.com/share/FQNYNAE6RDJ#/screens

**Comment by sarjon** [comment]
@LouiseBonnard seems that current wording for showcase card is already correct, right?

**Comment by Quetzacoalt91** [comment]
Hi @sarjon,

Shall we expect the assets to be generated before merging this one?

**Comment by sarjon** [comment]
Ummmm, I think no, there should be assets rebuild PR anyways, since not all PRs are merged with built assets.

But if you want, I can rebuild it. :)

**Comment by matks** [comment]
Thank you @sarjon

**Pull request merged**